### ISSUE TITLE
Broken renderer/Incorrect Debug Monitor readings for "Vertices Drawn" perf. counter

### ISSUE THREAD

**Issue by syskrank** [godotengine/godot#25957]
**Godot version:**

3.1 custom build, [962c14278](https://github.com/godotengine/godot/commit/962c14278fafecf3d047fd070fc7ab8d4dcc21da)

**OS/device including version:**
Linux Mint 19 x64
AMD RX570, proprietary drivers

**Issue description:**
When rendering just a single cube with GLES3 - built-in or imported as DAE from Blender, Debug monitor gives a reading of 72 vertices that are being rendered in one frame.

That's pointing to the possible problems either with Debug Monitor or maybe even with the renderer itself.
Is this number depicting not the number of vertices but the overall 3d data attributes count?

It's worth mentioning that GLES2 renderer does not report any statistics at all - every monitor is zeroed.

![image](https://user-images.githubusercontent.com/11526664/52905841-18b89300-3251-11e9-8484-9b7cfb32276b.png)

**Steps to reproduce:**
Create a simple scene with single cube and camera in it.
Start the application, switch to the Debugger/Monitors pane.
See that single cube takes 72 vertices to send and 2 draw calls to render.

**Minimal reproduction project:**
[3d_profiling.zip](https://github.com/godotengine/godot/files/2872419/3d_profiling.zip)

**Comment by syskrank** [comment]
Confirming the same thing for MacOS Mojave, i7, Intel UHD 630

**Comment by syskrank** [comment]
I used [apitrace](https://github.com/apitrace/apitrace) to investigate the rendering process for the simple cube scene.
Seems that the cube is fine:

![selection_035](https://user-images.githubusercontent.com/11526664/52912515-784f8680-32c3-11e9-8c2f-8fd491fdebc5.png)

We've got 24 vertices for cube 6 faces here? That's acceptable.
 
Although, there are 3 `glVertexAttribPointer` calls with 24 elements each, so I guess the Performance counter summarizes not the vertices drawn per frame but vertex attributes used per frame. Which is truly misleading.
